.github/workflows/nuclia_sync.yml

```diff
@@ -0,0 +1,30 @@
+name: Nuclia Sync
+
+on: [push]
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+
+    - name: Install requirements
+      run: |
+        pip install -q -r requirements-initial.txt
+        pip install -q -r requirements.txt
+    - name: Run Nuclia Sync

```

Comment:
Let's use consistent spacing between `name`s. I think a blank line between each pair is helpful to see each task.